### PR TITLE
crate release prep

### PR DESCRIPTION
**Pull request by 0xja**
Primarily, this PR splits out the NFT contract message structs into their own package called `collectxyz` for easier integration into other contracts that wish to call the xyz NFT contract. Additionally, it includes a couple small fixes and enhancements, which live in their own commits for easy review.

### CODE REVIEW

File: contracts/collectxyz-nft-contract/schema/xyz_tokens_response.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     },
     "Expiration": {
       "description": "Expiration represents a point in time when some event happens. It can compare with a BlockInfo and will return is_expired() == true once the condition is hit (and for every block in the future)",
-      "anyOf": [
+      "oneOf": [
```
Comment (collectxyznuisance) [inline contracts/collectxyz-nft-contract/schema/xyz_tokens_response.json:70]: Is this schema something you generate manually?

**Review by collectxyznuisance — APPROVED** [review]
Looks fantastic, @cocoloco80085 how much of a lift to update the planet contract?

What does it take to publish the crate now that we are ready for it?

File: contracts/collectxyz-nft-contract/schema/xyz_tokens_response.json (hunk 1 above)
Comment (0xja) [reply contracts/collectxyz-nft-contract/schema/xyz_tokens_response.json:70]: It's generated by running `cargo schema` from the NFT contract package root directory. The `schema` command executes this file: https://github.com/collectxyz/collectxyz-nft-contract/blob/0xja/crate-release-prep/contracts/collectxyz-nft-contract/examples/schema.rs

**Comment by cocoloco80085** [comment]
> Looks fantastic, @cocoloco80085 how much of a lift to update the planet contract?
> 
> What does it take to publish the crate now that we are ready for it?

minimal effort to integrate. Will need to create something similar to be used on the resources contract

**Comment by 0xja** [comment]
@collectxyznuisance should be as simple as running `cargo publish`, I believe.

**Pull request merged**